src/modules/layoutUiRules/rules.js

```diff
@@ -39,6 +39,22 @@ const defaultRules = {
     },
 }
 
+const dualCategoryRules = {
```

Comment:
Maybe the two category rules should be something that are applied on top of the vis type rules. So we could do e.g. `{ ...rulesByType, ...extraRulesByFeature }` so that we avoid `[VIS_TYPE_LINE]: dualCategoryRules`. Thoughts?

Reply:
Solved for now with the simplest solution.
As suggested by @martinkrulltott only Radar differs from the other vis types that used the default layout rules, so the new default rules are now for 2 category, and Radar got its own rules to only allow 1 dimension for categories.